### ISSUE TITLE
VisDial v0.9 evaluations

### ISSUE THREAD

**Issue by adnenabdessaied** [gicheonkang/gst-visdial#6]
Dear Gi-Cheon Kang,

I have a quick question about the results on the visdial v0.9 version. I took a look at the code and found out that you always started training from the 'basemodel' checkpoint of VisDial-Bert which was trained on VisDial v1.0.
As you know, the validation split of VisDial v0.9 is part of the training split of VisDial v1.0. Therefore, one cannot simply use the 'basemodel' checkpoint to train a model that will later be evaluated on VisDial v0.9.
Could you please clarify how you proceeded in training your model that was later evaluated on VisDial v0.9? As far as I know, VisDial-Bert was not trained on VisDial v0.9 and no appropriate checkpoints for this version of the dataset exist.

Thank you very much in advance

**Comment by gicheonkang** [comment]
Hello, thank you for your interests!

Yes, VisDial-Bert was not trained on VisDial v0.9. So, we trained our model on VisDial v0.9 dataset on top of the [VQA pretrained checkpoint](https://github.com/vmurahari3/visdial-bert/blob/master/scripts/download_checkpoints.sh) in the VisDial-Bert.

Consequently, you should use `-start_path vqa_pretrained_weights` when training on VisDial v0.9.

I forgot to mention it since we introduced all details based on VisDial v1.0.

Hope this helps. Thank you!

**Comment by adnenabdessaied** [comment]
Hello, 

Thank you for replying.
Could you please upload the checkpoint of this version of your model, i.e. the one that was trained on VisDial v0.9 and if possible the training and validation splits you used for training (both should be a subset of the train split of VisDial v0.9) as the Val split is kept for testing?

That will be very appreciated!

thanks

**Comment by gicheonkang** [comment]
Hi,
I just uploaded the checkpoints of the student model trained on v0.9. Here is the [link](https://www.dropbox.com/s/ldmw4ojgxrguukd/student_v0.9.ckpt?dl=0)!

What do you mean by training and validation split used for training? We used the entire VisDial v0.9 training dataset for training and evaluated the model on the VisDial v0.9 val data for testing. All hyperparameters are the same with v1.0 model which was tuned on the subset of v1.0 training data. VisDial v0.9 val data was just used for testing!

If I remember right, we used the [data for validation](https://www.dropbox.com/s/1ajjfpepzyt3q4m/visdial_1.0_train_dense_sample.json?dl=0) which is densely annotated labels in the VisDial v1.0 training data.

Feel free to ask if you have further questions.
Thanks
